.travis.yml

```
@@ -50,6 +50,9 @@ before_script:
   # Make sure Composer is up to date.
   - composer self-update
 
+  # Downgrade to composer version 1 to support Drupal 8.8.
+  - test ${TEST_SUITE} != "8.8.x" || composer self-update --1
```

This may be more readable:
```suggestion
  - if [ ${TEST_SUITE} == "8.8.x" ]; then composer self-update --1; fi
```